src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,146 @@
+package core.basesyntax.service;
+
+import core.basesyntax.InvalidDataException;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationServiceImpl registrationService;
+    private User defaultUser;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        Storage.people.clear();
+
+        defaultUser = new User();
+        defaultUser.setAge(18);
+        defaultUser.setLogin("username1");
+        defaultUser.setPassword("123456");
+    }
+
+    @Test
+    void register_nullAge_notOk() {
+        defaultUser.setAge(null);
+        Assertions.assertThrows(InvalidDataException.class,
+                () -> registrationService.register(defaultUser));
```

Comment:
I'd recommend to add static import for heavily used static method in order not to indicate each time class used
```suggestion
        assertThrows(InvalidDataException.class,
                () -> registrationService.register(defaultUser));
```
and
`import static org.junit.jupiter.api.Assertions.assertThrows;`